performance-tests/remotewb-performance-tests/src/main/resources/logback.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
AFAIK this jar is not published to any maven repository. These modules contain only tests. Therefore, I do not think that it is realistic that somebody would use this module as a dependency. ;-)